content/docs/user-guide/scripting/lua/_index.md

```diff
@@ -31,3 +37,33 @@ After you read through this tutorial on writing Lua scripts for the component en
+
+| Topic | Description |
+| --- | --- |
+| [Adding Lua Scripts to Component Entities](add-lua-script) | Use the Lua Script component to add script functionality to your game entities. |
+| [Lua Script Structure](basic-lua-script) | Basic structure of a Lua script. |
+| [Lua Editor](lua-editor) | Learn about the Lua Editor. |
+| [Properties Table](properties) | Specify the properties of a Lua script component that appear in O3DE Editor. |
+| [Using EBuses in Lua](ebus) | Write Lua scripts that use the EBus to communicate between components. |
+| [The Lua Environment (Advanced)](environment) | Learn about adding ScriptContext instances and using common code in the O3DE Lua environment. |
+| [Debugging Lua Scripts](debugging-scripts) | Learn about debugging Lua scripts in O3DE. |
+| [Debugging with Lua Editor](debugging-tutorial) | Use Lua Editor to debug Lua scripts. |
+
+## Related Topics
+
+| Topic | Description |
+| --- | --- |
+| [Camera Component](\docs\user-guide\components\reference\camera\camera) | Use the Camera component to allow an entity to be used as a camera. |
+| [Using Player Input](\docs\user-guide\interactivity\input\working-with-the-input-component) |  Work with the Input component in O3DE. |
+| [Input Component Event Bus Interface](\docs\user-guide\components\reference\gameplay\input-event-bus-interface) | Work with the Input component EBus (event bus). |
+| [Simple State Component](\docs\user-guide\components\reference\gameplay\simple-state) | Use the Simple State component to provide a simple state machine. |
+| [Tag Component](\docs\user-guide\components\reference\gameplay\tag) | Use the Tag component to apply labels to entities. |
+| [AWS Client Auth Scripting](\docs\user-guide\gems\reference\aws\aws-client-auth\scripting) | Examples of using the AWS Client Auth Gem with Script Canvas and Lua. |
+| [AWS Metrics Scripting](\docs\user-guide\gems\reference\aws\aws-metrics\scripting) | Examples of using Script Canvas or Lua with the AWS Metrics Gem to generate and submit metrics. |
+| [Gestures Gem](\docs\user-guide\gems\reference\input\gestures) | The Gestures Gem provides detection for common gesture-based input actions. |
+| [Virtual Gamepad Gem](\docs\user-guide\gems\reference\input\virtual-gamepad) | The Virtual Gamepad Gem provides controls that emulate a gamepad on touch screen devices for O3DE projects. |
+| [Tweener Lua Script](\docs\user-guide\interactivity\user-interface\animating\tweener-system\tweener-lua-code) | Learn how to use Lua scripting to animate your entities with the Scripted Entity Tweener system. |
+| [Tweener Timeline](\docs\user-guide\interactivity\user-interface\animating\tweener-system\tweener-timeline) | Use Scripted Entity Tweener's timeline abilities to chain animations together and exert fine control over them. |
+| [Synchronizing Animation Graphs](\docs\user-guide\visualization\animation\character-editor\sync-graph) | Use synchronized animation graphs to synchronize animation between actors. |
```

Comment:
Whoops.. these links must all use forward slash `/` in their path, not `\`.

Reply:
Thanks!